src/main/java/uk/gov/hmcts/reform/laubackend/cases/utils/InputParamsVerifier.java

```diff
@@ -66,8 +66,8 @@ public static void verifyRequestSearchParamsConditions(final SearchLog searchLog
         verifyUserId(searchLog.getUserId(), USERID_POST_EXCEPTION_MESSAGE);
         verifyTimestamp(searchLog.getTimestamp(), TIMESTAMP_POST_EXCEPTION_MESSAGE, TIMESTAMP_POST_REGEX);
         if (!CollectionUtils.isEmpty(searchLog.getCaseRefs())) {
-            for (String caseRef : searchLog.getCaseRefs()) {
-                verifyCaseRef(caseRef, CASEREF_POST_EXCEPTION_MESSAGE);
+            for (Long caseRef : searchLog.getCaseRefs()) {
```

Comment:
Again if we keep caseRefs as a String we don't need to change the code, just convert String to Long.